client/src/pages/SignUpForm.tsx

```diff
@@ -1,9 +1,14 @@
 import { Link, useNavigate } from 'react-router-dom';
 import { register } from '../lib/data';
+import { IoEyeSharp } from 'react-icons/io5';
 import { useState } from 'react';
+import { FaRegEyeSlash } from 'react-icons/fa6';
+
 function SignUpForm() {
   const navigate = useNavigate();
   const [isUsername, setIsUsername] = useState<boolean>(true);
+  const [pwdInputType, setPwdInputType] = useState<string>('password');
```

Comment:
Instead of using the type `string`, TypeScript could give you better error checking if you used `'text' | 'password'`, like this:
`useState<'text' | 'password'>('password');`